### ISSUE TITLE
Root ResourceDictionary without key in Application.Resources should not show as an error

### ISSUE THREAD

**Issue by Yasu3D** [ForNeVeR/AvaloniaRider#331]
When declaring a ResourceDictionary in Application.Resources like this:
```
<Application>
    <Application.Resources>
        <ResourceDictionary>
            
        </ResourceDictionary>
    </Application.Resources>
</Application
```

`<ResourceDictionary>` gets flagged as an error, because it doesn't contain an `x:Key=""` attribute.
Everything compiles fine, but the error is quite annoying and I haven't found a way to suppress it.

Not sure if this is an issue specific to AvaloniaRider, or if this is just Rider / Avalonia's fault, but I figured this would be a decent place to point it out.

**Comment by ForNeVeR** [comment]
Sorry, but you'll have to [report this to ReSharper](https://youtrack.jetbrains.com/issues/rsrp).

**Comment by Yasu3D** [comment]
Thanks for your time and for pointing me in the right direction!
[Seems this issue has already been reported there 5 years ago.](https://youtrack.jetbrains.com/issue/RSRP-471574/StyleSheet-tag-alongside-ResourceDictionary-causes-false-errors) Recently updated to "Fixed", though I'm not seeing any fix in v2023.3.3 yet. Best I can do is probably just wait.
